bin/luatest

```diff
@@ -1,4 +1,25 @@
 #!/usr/bin/env tarantool
 
-local runner = require('luatest').runner
-os.exit(runner:run())
+local fun = require('fun')
+
+-- Tarantool does not invoke Lua's GC-callbacks on exit.
+-- So this method clears every loaded package and invokes GC explicitly.
+-- It does not clear previously loaded packages to not break any GC-callback
+-- which relies on built-in package.
+local function gc_sandboxed(fn)
+    local original = fun.iter(package.loaded):map(function(x) return x, true end):tomap()
```

Comment:
It would be better if you cleaned loaded packages after every test file. So they don't interfere

Reply:
👎 for this. Also there can be common `test/helper.lua` file which is required before the whole suite.